services/src/contexts/session.rs

```diff
@@ -33,6 +33,14 @@ pub struct SimpleSession {
     id: SessionId,
     pub project: Option<ProjectId>,
     pub view: Option<STRectangle>,
+    #[serde(skip)]
```

Comment:
it's more about serialization. I didn't want to expose it via the API (because we don't need it and it doesn't fit to the UserSession). I don't think we deserialize Sessions.